ContactsApp/Exception.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Exception.swift
+//  ContactsApp
+//
+//  Created by Rohit Bansal on 04/10/15.
+//
+//
+
+import Foundation
+
+class Exception : NSObject, ErrorType {
```

Comment:
I don't think the use of Exception is needed in our case of validation.
http://c2.com/cgi/wiki?DontUseExceptionsForFlowControl

But in the case we needed ErrorType, they should be Domain Specific and use the Error prefix.
https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/ErrorHandling.html